orca-front50/src/test/groovy/com/netflix/spinnaker/orca/front50/tasks/SavePipelineTaskSpec.groovy

```diff
@@ -65,4 +65,67 @@ class SavePipelineTaskSpec extends Specification {
     ])
   }
 
+  def "should copy existing pipeline index to new pipeline"() {
+    Integer testIndex = 14
```

Comment:
From my non-existent knowledge of Spock conventions, I would put this (and `existingPipeline` in `when:`) in the `given:` block. Same for the next spec.